### PR TITLE
docs: Adding example for awaitMap()

### PR DESCRIPTION
**Pull request by arriolac**
Adding example how awaitMap() could be used.

### CODE REVIEW

**Comment by barbeau** [comment]
@googlebot I consent

File: app/src/main/java/com/google/maps/android/ktx/demo/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        // Shared object managers - used to support multiple layer types on the map simultaneously
+        val markerManager = MarkerManager(map)
+        val groundOverlayManager = GroundOverlayManager(map)
+        val polygonManager = PolygonManager(map)
+        val polylineManager = PolylineManager(map)
+
+        // Add clustering
+        val clusterManager = ClusterManager<MyItem>(this, map, markerManager)
+        map.setOnCameraIdleListener(clusterManager)
+        //addClusterItems(clusterManager)
```
Comment (barbeau) [inline app/src/main/java/com/google/maps/android/ktx/demo/MainActivity.kt]: @arriolac Did you intend to include the new demo app in this PR? It's still WIP.

Comment (arriolac) [reply app/src/main/java/com/google/maps/android/ktx/demo/MainActivity.kt]: Ah, definitely not! Meant to rebase before PR'ing... 😓

File: README.md [hunk 2]
```diff
@@ -59,6 +59,41 @@ val marker = googleMap.addMarker {
...
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    val mapFragment = supportFragmentManager.findFragmentById(R.id.map) as! SupportMapFragment
+
+    lifecycle.coroutineScope.launchWhenCreated {
+        val googleMap = mapFragment.awaitMap()
+    }
+}
+```
+
```
Comment (barbeau) [inline README.md:96]: It looks like GitHub won't let me add a suggestion that includes a code block (?), but I'd suggest something like this here:

```suggestion

Note that to use the [Lifecycle-aware coroutine scope][lifecycle] you'll need to add the following to your `build.gradle` dependencies:

`implementation 'androidx.lifecycle:lifecycle-runtime-ktx:<latest-version>'`

```

File: README.md [hunk 3]
```diff
@@ -59,6 +59,41 @@ val marker = googleMap.addMarker {
...
+_Before_
+```java
+@Override
+public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    SupportMapFragment mapFragment = ((SupportMapFragment) getSupportFragmentManager().findFragmentById(R.id.map));
+
+    mapFragment.getMapAsync(new OnMapReadyCallback {
+        @Override
+        public void onMapReady(GoogleMap map) {
```
Comment (barbeau) [inline README.md:77]: Suggest changing `map` to `googleMap` to match the Kotlin example:

```suggestion
        public void onMapReady(GoogleMap googleMap) {
```

**Pull request merged**